go/pkg/tool/tool.go

```diff
@@ -418,10 +422,21 @@ func (c *Client) DownloadAction(ctx context.Context, actionDigest, outputPath st
 			is.InputNodeProperties[path] = command.NodePropertiesFromAPI(t.NodeProperties)
 		}
 	}
-	if len(is.InputNodeProperties) == 0 {
-		return nil
+	if len(is.InputNodeProperties) != 0 {
+		err = c.writeProto(is, filepath.Join(outputPath, "input_node_properties.textproto"))
+		if err != nil {
+			return err
+		}
+	}
+	res, err := c.formatActionResStr(ctx, actionProto, resPb, commandProto, cmdDg)
+	if err != nil {
+		return fmt.Errorf("error showing action %v: %v", outputPath, err)
```

Comment:
Is the outputPath relevant in this error?
```suggestion
		return fmt.Errorf("error formatting action: %v", err)
```